src/braket/analog_hamiltonian_simulator/rydberg/rydberg_simulator_helpers.py

```diff
@@ -134,20 +137,22 @@ def _get_detuning_dict(
 
     detuning = {}  # The detuning term in the basis of configurations, as a dictionary
 
-    for ind_1, config in enumerate(configurations):
+    for config, ind_1 in configurations.items():
         value = sum([1 for ind_2, item in enumerate(config) if item == "r" and ind_2 in targets])
         if value > 0:
             detuning[(ind_1, ind_1)] = value
 
     return detuning
 
 
-def _get_rabi_dict(targets: Tuple[int], configurations: List[str]) -> Dict[Tuple[int, int], float]:
+def _get_rabi_dict(
+    targets: Tuple[int], configurations: Dict[str, int]
+) -> Dict[Tuple[int, int], float]:
     """Return the dict for the Rabi operators for a set of target atoms.
 
     Args:
         targets (Tuple[int]): The target atoms of the detuning operator
```

Comment:
It should be "Rabi operator", could you help me to correct this typo?

Reply:
Sorry meant to say in Line 154, the "detuning operator" should read "Rabi operator". That was my typo long time ago... :) could you help me fix it?